### ISSUE TITLE
brew install phantomjs fails on el capitan

### ISSUE THREAD

**Issue by chovy** [ariya/phantomjs#13754]
```
 brew install phantomjs
phantomjs: This formula either does not compile or function as expected on OS X
versions newer than Yosemite due to an upstream incompatibility.
Error: An unsatisfied requirement failed this build.
```

I tried downloading the 2.x binary for mac and did the example for `node-horseman` module and it doesn't appear to work:

```
var Horseman = require('node-horseman');
var horseman = new Horseman();

horseman
  .userAgent("Mozilla/5.0 (Windows NT 6.1; WOW64; rv:27.0) Gecko/20100101 Firefox/27.0")
  .open('http://www.google.com')
  .type('input[name="q"]', 'github')
  .click("button:contains('Google Search')")
  .keyboardEvent("keypress",16777221)
  .waitForSelector("div.g")
  .count("div.g")
  .log() // prints out the number of results
  .close();
```

**Comment by gordonleacock** [comment]
I'm in the same boat - get this error message. Any suggestions?

**Comment by ariya** [comment]
This is resolved in the next version 2.1.
